src/agent/onefuzz-task/src/tasks/fuzz/libfuzzer/common.rs

```diff
@@ -40,8 +40,14 @@ pub fn default_workers() -> usize {
     usize::max(1, cpus - 1)
 }
 
+pub trait LibFuzzerType {
```

Comment:
Will add doc comments.

I do think this is a bit awkward, but I'm trying to keep the changes here as minimal as possible. The `libfuzzer-dotnet` fuzzing task will really be extremely similar to the generic one, except for how we construct the fuzzer process environment and arguments.